pinot-server/src/test/java/org/apache/pinot/server/api/TablesResourceTest.java

```diff
@@ -273,6 +273,7 @@ public void testOfflineTableSegmentMetadata()
             .get(String.class));
     Assert.assertEquals(jsonResponse.get("columns").size(), 2);
     Assert.assertEquals(jsonResponse.get("indexes").size(), 17);
+    Assert.assertEquals(jsonResponse.get("star-tree-index").size(), 0);
```

Comment:
You might need to modify the test because it can be `null` here

Reply:
Fixed this by using `Assert.assertNull`

Reply:
Reverted this.